### PR TITLE
Feat: Grid component

### PR DESCRIPTION
**Pull request by pritam-kr**
- 1x2 grid layout
Screenshot (192)

- 1x3 grid layout

Screenshot (193)

### CODE REVIEW

File: components/Grid.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    padding: 0.5em;
+}
+
+.grid-img{
+     height: 100%;
+}
+
+.grid-3-wrapper{
+    display: grid;
+    grid-template-columns: 33% 33% 33%;
```
Comment (msdeshmukh009) [inline components/Grid.css:17]: You can also use `repeat(3,1fr);`

File: components/Grid.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    justify-content: center;
+}
+
+@media screen and (max-width: 876px) {
+    .grid-wrapper{
+        grid-template-columns: 100%;
+    }
+    .grid-3-wrapper{
+        grid-template-columns: 100%;
+        grid-template-rows: repeat(3, auto);
```
Comment (msdeshmukh009) [inline components/Grid.css:32]: I think you can skip this line as well. The browser will decide the number of rows implicitly. Not sure please tell me if am wrong.

**Review by msdeshmukh009 — APPROVED** [review]
Good work! Pritam.

**Review by srishti-maurya — APPROVED** [review]
looks good

File: components/Grid.css (hunk 2 above)
Comment (pritam-kr) [reply components/Grid.css:32]: it decides. Thank you for review

**Pull request merged**